trinity/config.py

```diff
@@ -169,19 +183,49 @@ def data_dir(self) -> Path:
     def data_dir(self, value: str) -> None:
         self._data_dir = Path(value).resolve()
 
+    @property
+    def is_db_backend_leveldb(self) -> bool:
+        return self.db_backend == DB_LEVEL
+
+    @property
+    def is_db_backend_rocksdb(self) -> bool:
+        return self.db_backend == DB_ROCKS
```

Comment:
I want to say YAGNI about these `is_db_b`... properties, and go back to just running these lines in the if test internally. Do we lose much by dropping the methods?

Reply:
I wrote these because I don't want to have to import the `DB_ROCKS` and `DB_LEVEL` constants in the places that these checks happen.  I ended up doing the same with `is_full_sync` and `is_fast_sync`.  Unless you feel strongly I'd like to leave them.